plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -42,6 +43,7 @@ public static JSONObject generateSchema() {
                 BaseConfigurator baseConfigurator = (BaseConfigurator) configuratorObject;
                 List<Attribute> baseConfigAttributeList = baseConfigurator.getAttributes();
                 if (baseConfigAttributeList.size() == 0) {
+                    System.out.println("Help file" + Jenkins.getInstance().getDescriptor(((BaseConfigurator) configuratorObject).getTarget()).getHelpFile());
```

Comment:
@timja I tried debugging but this `get-descriptor` returns an NPE.

Reply:
I wouldn't expect it to, you need to be in the HeteroDescribeableConfigurator I believe

Reply:
Yeah would it not. Make sense to pass the field name from here so that we get the help function returned?

Reply:
this is the code for looking up the descriptor if you need it somewhere else:
https://github.com/jenkinsci/configuration-as-code-plugin/blob/81b9a6857b03307d77bb58bdfa2e348c8ee4eed3/plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/HeteroDescribableConfigurator.java#L205

Reply:
We need to get the descriptor by just passing the configurator object, much like how its done here.Unless there is a way to get the symbol and Cnode from the base configurator.